src/utils.py

```diff
@@ -57,9 +58,11 @@ def get_dataset(args):
                                       transform=apply_transform)
 
         # sample training data amongst users
-        if args.iid:
+        if args.iid == 1:
             # Sample IID user data from Mnist
             user_groups = mnist_iid(train_dataset, args.num_users)
+        elif args.iid == 2:
```

Comment:
Can you enable us to set the beta value as a parameter when calling it from the terminal (as an argument)?

Reply:
done